src/lib/alarmMgr/AlarmManager.cpp

```diff
@@ -37,20 +37,13 @@
 
 /* ****************************************************************************
 *
-* AlarmManager::AlarmManager - 
+* badInputSent - 
```

Comment:
As we were discussing at skype "sent" could be missleading, as alarms are not actually send (in the way a notification or a forwarderd request is sent). Alarms are "raised" and "released", but not "sent".

Looking to how this variable is used in the code, maybe "badInputAlreadySeen" or just "badInputSeen" could be a better name.